### ISSUE TITLE
can not select video/photos from library

### ISSUE THREAD

**Issue by Sumrocks** [marcshilling/react-native-image-picker#436]
looks like callback is not work
no print!!! 
same to 'launchCamera';
version ："react-native-image-picker": "^0.24.0", 

```
const videoOptions = {
    mediaType: 'video',
    videoQuality: 'high',
    durationLimit: 15,
};
RNImagePicker.launchImageLibrary(videoOptions,(response)=>{
       console.log(response);
       let photos = [];
       let videoUri = {}
       if (response.uri) {
            videoUri.uri = response.uri
            photos.push(videoUri);
       }
       //this.chooseToResponse(photos);
});

```

**Comment by Sumrocks** [comment]
forgot something.
It works well in pure reactNative project, when i move to android project, troubles coming. 
can you give some suggestion

**Comment by Sumrocks** [comment]
Solved! ==>#383
@Override public void onActivityResult(int requestCode, int resultCode, Intent data) {
       super.onActivityResult(requestCode, resultCode, data);
       mReactInstanceManager.onActivityResult(this, requestCode, resultCode, data) ;
}
